src/nvim/drawline.c

```diff
@@ -1217,7 +1315,23 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool nochange,
         }
       }
 
-      if (draw_state == WL_NR && n_extra == 0) {
+      if (draw_state == WL_STC - 1 && n_extra == 0) {
+        draw_state = WL_STC;
+        // Draw the 'statuscolumn' if option is set.
+        if (statuscol.draw) {
+          if (statuscol.text_len == 0) {
+            get_statuscol_str(wp, lnum, row, startrow, filler_lines, cul_attr,
+                              sign_num_attr, sattrs, foldinfo, extra, &statuscol);
+            if (wp->w_redr_statuscol) {
+              return 0;
+            }
+          }
+          get_statuscol_display_info(&draw_state, &char_attr, &n_extra, &c_extra,
+                                     &c_final, extra, &p_extra, &statuscol);
+        }
+      }
+
+      if (draw_state == WL_BRI - 1 && n_extra == 0) {
```

Comment:
```suggestion
      if (draw_state == WL_STC && n_extra == 0) {
```

Reply:
All instances of `WL_BRI - 1` or just that one?

Reply:
Just this one.